packages/blockchain-wallet-v4-frontend/src/data/analytics/types/onboardingAndVerification.ts

```diff
@@ -0,0 +1,264 @@
+// Onboarding and Verification Events
+export enum Events {
+  ONBOARDING_ADDRESS_ENTERED = 'Address Information Entered',
```

Comment:
Not a show stopper, but I wonder if we really need these event keys as an enum? Since they're only ever used as the key to a specific type couldn't we save some lines of code by removing the enum and just putting the key string directly in the event type? 
ex.
```
type EmailVerificationRequestAction = {
  key: 'Email Verification Requested'
  properties: {
    origin: OriginProductFlow
  }
}
```
It should still be type safe and we should still have autocomplete I believe.

Reply:
Actually we use them when we cll event itself from any location, we can do it like you said but with current implementation we use enum and then you can use it like `AnalyticsKey.PEEKSHEET_SELECTION_CLICKED` anywhere in the app rather than to use just strings around.